test/extended/testdata/imagestream-jenkins-slave-pods.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: ImageStream
+metadata:
+  annotations:
+    slave-label: asdf
+  labels:
+    role: jenkins-slave
+  name: asdf
+spec:
+  tags:
+  - annotations:
+      role: jenkins-slave
+    from:
+      kind: DockerImage
+      name: docker.io/openshift/jenkins-slave-maven-centos7:latest
+    name: base
+  - annotations:
+      role: jenkins-slave
+    from:
+      kind: ImageStreamTag
+      name: base
+    name: latest
```

Comment:
I was trying to combine things during my foray into bindata.go size.  Circling back I suspect the additions are small enough that they can be split out.